lib/plugins/cleaner.rb

```diff
@@ -0,0 +1,51 @@
+require 'rubygems/package'
+require_relative '../moonshot/creds_helper'
+
+module Moonshot
+  module Plugins
+    # Moonshot plugin class for removing artifacts when a stage gets deleted.
+    class Cleaner
+      include Moonshot::CredsHelper
+
+      def initialize(bucket_name)
+        @bucket_name = bucket_name
+      end
+
+      def post_delete(resources)
+        stack_name = resources.stack.name
+        resources.ilog.start 'Starting to delete artifacts for stack: ' \
```

Comment:
If you include `ResourcesHelper`, you get a reader method called `ilog` that could be used to reduce the length of this line without sacrificing readability as well.